R/checks.R

```diff
@@ -1,3 +1,43 @@
+# Additional type checkers designed for testing argument values.
+
+check_single_logical <- function(x, call = rlang::caller_env()) {
```

Comment:
Unfortunately, `rlang::arg_match()` only works with character inputs.

Reply:
I believe you're looking for `check_bool()` here!

Reply:
God I wish these were documented better.